lib/libzfs/os/freebsd/libzfs_zmount.c

```diff
@@ -116,7 +116,7 @@ do_mount(zfs_handle_t *zhp, const char *mntpt, const char *opts, int flags)
 {
 
 	return (do_mount_(zfs_get_name(zhp), mntpt, flags, NULL, 0,
-	    opts, sizeof (mntpt)));
+	    opts, MNT_LINE_MAX));
```

Comment:
Really seems like `do_mount_()` doesn't even need to be a separate function.

Reply:
Good point. I will combine them and repush when I get a chance.

Reply:
I have combined them and repushed.